config/settings.toml

```diff
@@ -55,3 +55,6 @@ _remove_workspace = true
 # that ansible-playbook accepts for the --extra-vars parameter
 # See https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#defining-variables-at-runtime  # noqa
 ansible_extra_vars = ""
+
+# registry cache IP
+registry_mirror_address = "192.168.122.1"
```

Comment:
Maybe it is better to explain in comments what is the default 192.168.122.1, maybe it should be the IP address of the bridge at your host machine, so it should correspond to what is provided via `network_range`?

Reply:
Another question, does this change makes the *registry_mirror_address* mandatory?

Reply:
Added clearer doc statements